integrations/FacebookPixel/constants.js

```diff
@@ -6,4 +6,72 @@ const CNameMapping = {
   FB_PIXEL: NAME,
 };
 
-export { NAME, CNameMapping };
+const traitsMapper = [
+  {
+    destKey: "external_id",
+    sourceKeys: [
+      "userId",
+      "traits.userId",
+      "traits.id",
+      "context.traits.userId",
+      "context.traits.id",
+      "anonymousId",
+    ],
+  },
+  {
+    destKey: "em",
+    sourceKeys: ["traits.email", "context.traits.email"],
+  },
```

Comment:
Previously for web device mode we used to fetch data from `context.traits` field only. We have changed the order of priority here. Let us check and be confirmed.

Reply:
@ItsSudip we can omit `traits` JS sdk only can expect `context.traits`  in identify.